tests/rustdoc-gui/sidebar-mobile.goml

```diff
@@ -72,6 +73,8 @@ define-function: (
             "background-color": |background|,
             "color": |color|,
         })
+        // Close the sidebar menu.
+        press-key: "Escape"
     },
 )
 
```

Comment:
If you could just check that the sidebar has the same width as the `<body>` (using [compare-elements-size](https://github.com/GuillaumeGomez/browser-UI-test/blob/master/goml-script.md#compare-elements-size), it'd be perfect.